components/Tabs/Tabs.js

```diff
@@ -1,38 +1,39 @@
 
 class TabsItem {
   constructor(element) {
-    // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
 
   select() {
-    // should use classList
+    this.element.classList.add("Tabs__item-selected");
   }
 
   deselect() {
-    // should use classList
+    this.element.classList.remove("Tabs__item-selected");
   }
 }
 
 class TabsLink {
-  constructor(element, parent) {
-    this.element;// attach dom element to object
-    this.tabs;// attach parent to object
-    this.tabsItem;// assign this to the associated tab using the parent's "getTab" method by passing it the correct data
-    // reassign this.tabsItem to be a new instance of TabsItem, passing it this.tabsItem
-    this.element.addEventListener('click', () => {
-      this.tabs.updateActive(this);
-      this.select();
-    });
+  constructor(element) {
+    this.element = element;
+  //   console.log(this.element);
```

Comment:
Love seeing you comment out your prior work with comments. Keep it up! When I was programming, I found it to be extremely helpful in making sure I did not revert back to my mistakes haha.